src/qcodes/instrument_drivers/rohde_schwarz/ZNLE.py

```diff
@@ -0,0 +1,4 @@
+from .ZNB import RohdeSchwarzZNBBase
+
+class ZNLE(RohdeSchwarzZNBBase):
+    pass
```

Comment:
These short names are soft deprecated so let's not add them for new instruments.

Rather please create classes of the form RohdeSchwarzZNLE3 in a private file `_rohde_schwarz_znle.py` 
and add the imports of these files to the `__init__.py` file in the folder.